src/utils/utils.service.js

```diff
@@ -59,6 +59,47 @@ class UtilsService {
       return index
     }, {})
   }
+
+  checkAndResolveConflicts (changedDoc, pouchdb) {
+    let preferredRevision = {}
+    if (!changedDoc._conflicts) {
+      return
+    }
+
+    if (!changedDoc.updatedAt) {
+      preferredRevision = changedDoc
+      changedDoc._conflicts.map((conflictingRev) => {
+        if (conflictingRev.updatedAt) {
+          preferredRevision = conflictingRev
+        } else {
+          pouchdb.remove(changedDoc._id, conflictingRev._rev)
+        }
+      })
+    } else {
+      let serializedRevisions = this.serialiseDocWithConflictsByProp(changedDoc, 'updatedAt')
```

Comment:
Yes. also adds the main doc (couchdb's "winning" rev) so all are in an array and can be sorted